src/model/modifier/AtomicUtils.js

```diff
@@ -27,8 +27,8 @@ const {
   Repeat,
 } = Immutable;
 
-const MediaUtils = {
-  insertMedia: function(
+const AtomicUtils = {
```

Comment:
Maybe `AtomicBlockUtils`, `insertAtomicBlock`, etc?

I think the term "media" stands on its own a bit better than "atomic". The block type is fine as `'atomic'`, I think, but elsewhere it may be worthwhile to provide a bit more clarity in naming.